pkg/models/models.go

```diff
@@ -62,61 +60,53 @@ type GameMapService interface {
 	Create(seed int64) GameMap
 }
 
-
 /* -------- CHARACTERS -------- */
 
 type Element int
 
 const (
 	Normal Element = iota
-	Fire  
-	Water  
-	Earth  
-	Air 
+	Fire
+	Water
+	Earth
+	Air
 )
 
 type Field int
 
 const (
-	Health Field = iota // from 10 to 100
-	Stamina // from 0 to 10
-	Speed // from 0 to 10
-	Critical // from 0 to 100
-	CriticalDamage // from 0 to 100
-	Dodge // from 0 to 100
-	HandSize // from 0 to 10
+	Health         Field = iota // from 10 to 100
+	Stamina                     // from 0 to 10
+	Speed                       // from 0 to 10
+	Critical                    // from 0 to 100
+	CriticalDamage              // from 0 to 100
+	Dodge                       // from 0 to 100
+	HandSize                    // from 0 to 10
 )
 
 type Character struct {
-	ID int 
-	Name string 
-	Type Element
-	
-	Parameters map[Field]int
-	
-	Deck []Card
-    Hand []Card 
+	Parameters  map[Field]int
+	Name        string
+	Deck        []Card
+	Hand        []Card
 	CardsPatern []int
-	Discard []Card
-
-	Curses []Effect 
-	Buffs []Effect 
-	// Items []Item
-
-	IsDead bool
+	Discard     []Card
+	Curses      []Effect
+	Buffs       []Effect
+	ID          int
+	Type        Element
+	IsDead      bool
 }
 
-
 /* -------- CARDS -------- */
 
 type Card struct {
-	ID int 
-	Name string 
-	Description string 
-	Cost int
-	Rarity Rarity
-
-	Actions []IAction
+	Name        string
```

Comment:
why change properties order ?

Reply:
Using the following tool : https://pkg.go.dev/golang.org/x/tools/go/analysis/passes/fieldalignment 

The order of the fields is important to optimize the size of the struct